Automattic/wp-calypso
# Site monitoring: Allow wrapping lines on date and timestamp columns

- Closes https://github.com/Automattic/dotcom-forge/issues/3833

## Proposed Changes

* Allow displaying date column in multiple lines to avoid overlap with next column.
* Removes the word `on` for languages different than English.

## Testing Instructions



* Change your preferred language to `spanish` on Account settings https://wordpress.com/me/account
* On an atomic site, navigate to `/site-monitoring/{urlSlug}`.
* Click on the `Web Server Logs`.
* Observe the date is displayed in two lines instead of overlapping.
* Observe there is a `,` on the date instead of displaying the word `on`.
* Change your account language to English.
* Observe the word `on` is present in the date.
* Click on `PHP logs` and repeat the testing instructions.

| **Before** | **After** |
|---|---|
| <img width="1266" alt="Screenshot 2023-09-20 at 17 00 50" src="https://github.com/Automattic/wp-calypso/assets/779993/51115ff5-a083-4823-8914-583af9614d3c"> | <img width="1264" alt="Screenshot 2023-09-20 at 17 01 16" src="https://github.com/Automattic/wp-calypso/assets/779993/725bd5fa-1170-4773-844d-66318cc47c0e"> |


## Pre-merge Checklist
